.github/workflows/regression-tests.yml

```diff
@@ -168,6 +168,17 @@ jobs:
             cc: clang
             extra_name: "clang | "
 
+          - sim: icarus             # with gcov/gcovr
+            sim-version: v10_3
+            lang: verilog
+            python-version: 3.8
+            os: ubuntu-latest
+            extra_name: "gcovr | "
+            cxx: ccache g++
+            cc: ccache gcc
+            CFLAGS: -g --coverage -fprofile-arcs -ftest-coverage
```

Comment:
`--coverage` is a shortcut for `-fprofile-arcs -ftest-coverage`, you don't need both.